pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -144,6 +147,12 @@ type Options struct {
 
 	// If meshConfig.DiscoverySelectors are specified, the DiscoveryNamespacesFilter tracks the namespaces this controller watches.
 	DiscoveryNamespacesFilter filter.DiscoveryNamespacesFilter
+
+	// EnableCRDScan determines whether the controller will list all CRDs
+	// present in the cluster, and subsequently only create watches on those
+	// that are. If this is set to false, all CRDs defined in the schema must be
+	// present for istiod to function.
+	EnableCRDScan bool
```

Comment:
This addition seems misplaced. It should be in the commit `[no-cluster-rbac] MAISTRA-1895: Add option to enable/disable CRD scan (#260) (#546)`

Reply:
Done